src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Sidenav from './component/sidenav/sidenav';
+import Events from './component/pages/events';
+import Reservation from './component/pages/reservation';
+import Reserve from './component/pages/reserve';
+import CreateEvent from './component/pages/new_event';
+import DeleteEvent from './component/pages/delete_event';
+import Login from './component/pages/login';
+import Signup from './component/pages/signup';
+import EventsDetails from './component/pages/events_details';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

![image](https://user-images.githubusercontent.com/57421823/213193132-aeb9e044-6303-4563-bd94-5c04ae4d6580.png)